app/process/handler/crypto-transfer/crypto-transfer_test.go

```diff
@@ -0,0 +1,192 @@
+package crypto_transfer
```

Comment:
- lets change the package to `cryptotransfer` for both the logic file and test file
- We could add unit tests for the `checkForTransactionCompletion` function as-well